src/pymovements/gaze/io.py

```diff
@@ -158,3 +158,59 @@ def from_csv(
         acceleration_columns=acceleration_columns,
     )
     return gaze_df
+
+
+def from_ipc(
+        file: str | Path,
+        experiment: Experiment | None = None,
+        **read_csv_kwargs: Any,
+) -> GazeDataFrame:
+    """Initialize a :py:class:`pymovements.gaze.gaze_dataframe.GazeDataFrame`.
+
+    Parameters
+    ----------
+    file:
+        Path of IPC/feather file.
+    experiment : Experiment
+        The experiment definition.
+    **read_csv_kwargs:
+            Additional keyword arguments to be passed to polars to read in the csv.
+
+    Examples
+    --------
+    First let's assume a IPC file stored `tests/gaze/io/files/monocular_example.feather`
+
+    We can now load the data into a ``GazeDataFrame`` by specyfing the experimental setting
```

Comment:
this sentence needs to be update, the experimental setting is not specified in this example

Reply:
k